zvariant/src/signature_parser.rs

```diff
@@ -30,6 +30,25 @@ impl<'s> SignatureParser<'s> {
         }
     }
 
+    pub fn from_bytes(signature: &'s [u8]) -> Result<Self> {
```

Comment:
I think the whole method should be marked as unsafe as the safety here depends on the caller. Only thing this method is checking is the length of the signature.

Reply:
ok, actually we don't need to make it pub either (well it's internal anyway, right)

Reply:
ok